### PR TITLE
Not including the vlan ports in case the VLAN is '-'

### PR DESCRIPTION
**Pull request by farhan7500**

### CODE REVIEW

**Comment by farhan7500** [comment]
@nilangekarss @wdurairaj Please review and merge

File: ansible_3par_docker_plugin/tasks/create_conf_file.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@
     no_log: True
 
   - name: get the iscsi ports (vlan tagged)
-    local_action: shell /usr/bin/sshpass -p {{ INVENTORY[item]['hpe3par_password'] }} ssh -oStrictHostKeyChecking=no {{ INVENTORY[item]['hpe3par_username'] }}@{{ INVENTORY[item]['hpe3par_ip'] }} "showport -iscsivlans" | awk '{print $3}' | sed -n '1!p'
```
Comment (farhan7500) [inline ansible_3par_docker_plugin/tasks/create_conf_file.yml]: In the getports -iscsivlans response, if the VLAN is not set (set to '-'), we ignore that as they are unresponsive (ping fails to connect)

File: ansible_3par_docker_plugin/tasks/create_conf_file.yml [hunk 2]
```diff
@@ -164,7 +164,7 @@
     no_log: True
 
   - name: get the iscsi ports of replication device (vlan tagged)
-    local_action: shell /usr/bin/sshpass -p {{ INVENTORY[item]['replication_device']['hpe3par_password'] }} ssh -oStrictHostKeyChecking=no {{ INVENTORY[item]['replication_device']['hpe3par_username'] }}@{{ INVENTORY[item]['replication_device']['hpe3par_ip'] }} "showport -iscsivlans" | awk '{print $3}' | sed -n '1!p'
+    local_action: shell /usr/bin/sshpass -p {{ INVENTORY[item]['replication_device']['hpe3par_password'] }} ssh -oStrictHostKeyChecking=no {{ INVENTORY[item]['replication_device']['hpe3par_username'] }}@{{ INVENTORY[item]['replication_device']['hpe3par_ip'] }} "showport -iscsivlans" | grep -v '-' | awk '{print $3}' | sed -n '1!p'
```
Comment (farhan7500) [inline ansible_3par_docker_plugin/tasks/create_conf_file.yml]: Same as above

**Comment by prablr79** [comment]
@nilangekarss any more changes required? can this be merged ?

**Comment by farhan7500** [comment]
@prablr79 @prablr79 No more changes for this. This PR is ready for merge, barring any review comments.

**Pull request merged**